public/index.html

```diff
@@ -126,7 +126,7 @@ <h3 id="cool">/cool/:from</h3>
 
         <h3 id="field">/field/:name/:from/:reference</h3>
```

Comment:
I think that this is the line that needs to change; it doesn't make sense for `:from` to be the recipient of the message.